string[] jointIndex
float64[] incrementAngle
---
int8 result